doc/semitrans.xml

```diff
@@ -251,3 +251,83 @@ Transformation( [ 7, 8, 8, 7, 8, 8, 8, 7, 8, 8 ] )]]></Example>
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DigraphOfActionOnPairs">
+  <ManSection>
+    <Attr Name = "DigraphOfActionOnPairs" Arg = "coll" Label = "for ever" />
+    <Oper Name = "DigraphOfActionOnPairs" Arg = "coll[, n]" Label = "never" />
```

Comment:
The label's here look weird. These will be visible in the index, so could they be more descriptive? "for a transformation collection" and "for a transformation collection and positive integer"